server/models/groups.js

```diff
@@ -0,0 +1,57 @@
+const { verifyAccessToken } = require('../utils/token-helpers');
+const { HttpError } = require('../utils/errors');
+const prisma = require('../db');
+
+class Group {
+  constructor(name, token) {
```

Comment:
```suggestion
  constructor(name,id) {
```

I think we shouldn't couple the group and the token verification logic.  We shouldn't add that dependency.  
Let the group take in `id` explicitly

Reply:
This was an eye opener, hadn't seen the coupling, thanks